### ISSUE TITLE
main menu carets, dropdowns not showing

### ISSUE THREAD

**Issue by peterm95018** [SU-SWS/open_framework#78]
I recently upgraded to 2.02. I'm not seeing drop downs show up in my main menu. Did the typical cache clear. Anything else I should be looking for in my config?

thanks,

Peter

**Comment by hyperboy** [comment]
The drop downs in the main menu now require a mouse-click instead of a mouse-over.  Users had trouble getting the mouse-over drop downs to appear on the tablet.  I'll add this update to the change log.

Thank you for your feedback.  We'll look at making the main menu mouse-over drop downs a theme option in the future.  You may want to revert back to version 2.01 in the meanwhile to retain the mouse-over drop downs.

**Comment by peterm95018** [comment]
Ok. But, I'm not sure I was clear enough in my original description. 

I thought I should be seeing carets as a visual indicator for those main menus with child pages. Take a look at http://maps-dev-d7.ucsc.edu. The Places, Directions, Printable Maps and Quick Links should all show carets. If you click on one, we get the expected behavior for that menu (but not the others). 

Thanks for the help and work on this project.

**Comment by hyperboy** [comment]
Thank you for the clarification and using our theme for your maps project!  You've probably loaded a menu block into the "Navigation" region.  You'll need to update your menu link settings to enable the display of the drop down menus.

1) Go to Administration > Structure > Menus > Main menu (or the menu you've used)
2) Click "edit" for the top level links like Places, Directions, Printable Maps and Quick Links
3) Make sure the check box is checked for "Show as expanded" -> Save

The interesting thing is that if you simply load the "Main menu" as is without using the "Navigation" region, it ignores the "Show as expanded" check box setting and shows all nested menu links as drop downs.  The "Navigation" region is intended for situations like sub-sites when it's necessary to override the default Main menu links.  Hope that helps.

**Comment by hyperboy** [comment]
Additional note:

By selecting the "Show as expanded", you lose access to the top level pages.  The "Show as expanded" menu links turn into a pure toggle for showing and hiding the drop down menus.  You may want to create additional drop down menu links like "Overview" or "Directions to Campus" for linking to the top level pages.

**Comment by peterm95018** [comment]
Voila! Thanks for the help and yes, it's time to re-work some of these menus and landing pages to be more cohesive. I keep switching things around after reading Mobile First. I'm trying to keep the best of the prod site (D6), but develop for a lost freshmen in the woods... :)
